pulpcore/pulpcore/app/serializers/status.py

```diff
@@ -49,22 +49,22 @@ class StatusSerializer(serializers.Serializer):
     )
 
     online_workers = WorkerSerializer(
-        help_text=_("List of online celery workers known to the application. An online worker is "
-                    "actively heartbeating and can respond to new work"),
+        help_text=_("List of online workers known to the application. An online worker is actively "
+                    "heartbeating and can respond to new work"),
         many=True
     )
 
     missing_workers = WorkerSerializer(
-        help_text=_("List of missing celery workers known to the application. A missing worker is "
-                    "a worker that was online, but has now stopped heartbeating and has "
-                    "potentially died"),
+        help_text=_("List of missing workers known to the application. A missing worker is a "
+                    "worker that was online, but has now stopped heartbeating and has potentially "
+                    "died"),
         many=True
     )
 
     database_connection = DatabaseConnectionSerializer(
         help_text=_("Database connection information")
     )
 
-    messaging_connection = MessageBrokerConnectionSerializer(
-        help_text=_("Message broker connection information")
+    redis_connection = RedisConnectionSerializer(
```

Comment:
This seems like it's exposing an implementation detail. Maybe make this more generic like `queue_connection` or something? This would cover us in the event that Oracle acquires Redis Labs and the community creates a fork of redis.